src/sass/index.sass

```diff
@@ -123,20 +126,22 @@
   #pause
     display: none
   
+  #menu
+    top: auto !important
+
   .window-app
     display: flex !important
     flex-direction: column !important
     width: 100% !important
     min-width: 0 !important
     min-height: 0 !important
-    top: 0 !important
+    top: auto !important
     left: 0 !important
     height: auto !important
-    max-height: calc(100vh - var(--hotbar-height)) !important
+    max-height: 70vh !important
     overflow-y: hidden !important
 
-    .configure-sheet,
-    .configure-token,
+    .header-button:not(.close),
```

Comment:
I had some other addons that added header buttons so the title bar still showed some buttons